### DISCUSSION TITLE
Sidemenu missing in System

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#32544] [General]
The side menu is missing in system and so the administration of a site has really bad usability. A option to switch to template styles for example is needed here. Also the other items should offer the possibility to switch to the other areas of configuration.

<img width="1438" alt="grafik" src="https://user-images.githubusercontent.com/828371/109397220-fbd1f280-7935-11eb-8566-aece0c37df8b.png">

**Comment by Quy** [comment]
Please see #26409

**Comment by dgrammatiko** [comment]
> A option to switch to template styles for example is needed here

The problem is not the missing sidemenu but the really badly coded com_templates: https://github.com/joomla/joomla-cms/pull/30359

**Reply by coolcat-creations** [reply]
Agree. How can I help on your concept in #30359 ?

**Reply by coolcat-creations** [reply]
+ Same with language overides ;(

**Reply by dgrammatiko** [reply]
> How can I help on your concept

If you can create some rough mockups of the list view. My problem is that I can't decide how to layout the data (should be both the template and the styles somehow combined). 
- The template could have a dropdown with actions like Create Child, Manage Template Files, Manage Overrides, Manage Static Assets
- The styles could also have a dropdown with actions like Duplicate, Delete, Edit, Make default

I might have missed some actions but that's ok, at this point we're looking at the overall layout, the rest is just implementation details

**Reply by brianteeman** [reply]
What is wrong with language overrides. Unless I am missing something you can switch between site/admin languages and english/german etc without returning to the system dashboard - just like in j3.

I get what you mean about templates which is why I supported 30359 but I dont see what it missing with language overrides.

**Reply by brianteeman** [reply]
@dgrammatiko why not just go with the layout from that other template

**Reply by coolcat-creations** [reply]
> What is wrong with language overrides. Unless I am missing something you can switch between site/admin languages and english/german etc without returning to the system dashboard - just like in j3.
> 
> I get what you mean about templates which is why I supported 30359 but I dont see what it missing with language overrides.


They follow a completely different Ui logic than any other Component. Normally you switch between the context site/admin and have search tools and the language it set inside the item. Language overrides behave completely different and are not logical to someone who just learned how Joomla works. You also cant compare the overrides from different languages in one list. They are not broken in comparsion to J3 but they are not consistent and good to handle.

**Reply by coolcat-creations** [reply]
> @dgrammatiko why not just go with the layout from that other template

I think if there has been done good work by khonsu, why not taking good concepts into J4 :-) If you need any changes, then tell me and I provide changed mockups if needed.

**Reply by dgrammatiko** [reply]
it was still based on the concept of templates/styles, eg 2 independent unrelated lists, iirc. I think I checked some time ago but maybe I'm wrong. Will check again

**Reply by brianteeman** [reply]
The issue with Language overrides is, while perhaps valid, is completely different to a missing sidemenu.

**Reply by brianteeman** [reply]
You already have an open issue for that part of language overrides https://github.com/joomla/joomla-cms/issues/28855
